onnxruntime/test/perftest/ort_test_session.cc

```diff
@@ -323,6 +323,55 @@ OnnxRuntimeTestSession::OnnxRuntimeTestSession(Ort::Env& env, std::random_device
 #endif
   } else if (provider_name == onnxruntime::kNnapiExecutionProvider) {
 #ifdef USE_NNAPI
+    uint32_t nnapi_flags = 0;
+#ifdef _MSC_VER
+    std::string ov_string = ToMBString(performance_test_config.run_config.ep_runtime_config_string);
+#else
+    std::string ov_string = performance_test_config.run_config.ep_runtime_config_string;
+#endif
+    std::istringstream ss(ov_string);
+    std::string token;
+    while (ss >> token) {
+      if (token == "") {
+        continue;
+      }
+      auto pos = token.find("|");
+      if (pos == std::string::npos || pos == 0 || pos == token.length()) {
+        ORT_THROW("[ERROR] [TensorRT] Use a '|' to separate the key and value for the run-time option you are trying to use.\n");
+      }
+
+      auto key = token.substr(0, pos);
+      auto value = token.substr(pos + 1);
+      if (key == "NNAPI_FLAG_USE_FP16") {
```

Comment:
These options are by default disabled, you don't need to specify true or false for it, you can simply specify which option to be used and separate them by space, such as 
```
-i "NNAPI_FLAG_CPU_DISABLED NNAPI_FLAG_USE_NCHW"
```

Reply:
Yes, It could be. But I think giving a explicit value "true/false" is more straightforward. It's also easy to be applied once we the default value is changed someday.

Reply:
The NNAPI flag is only enabled when it is specified (flip the bit to 1), the default value is always 0

Reply:
fixed